x-pack/plugins/observability/server/services/slo/find_slo.ts

```diff
@@ -71,5 +81,12 @@ function toPagination(params: FindSLOParams): Pagination {
 }
 
 function toCriteria(params: FindSLOParams): Criteria {
-  return { name: params.name };
+  return { name: params.name, indicatorType: params.indicator_type };
+}
+
+function toSort(params: FindSLOParams): Sort {
+  return {
+    field: params.sort_by === 'indicator_type' ? SORT_FIELD.IndicatorType : SORT_FIELD.Name,
```

Comment:
Since we have only two possibilities at the moment, with Name being the default, I used a ternary operator for now